### PR TITLE
Add conferences from Whitehall media and other related to cybersecurity

### PR DESCRIPTION
**Pull request by mikamakusa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -256,11 +262,13 @@ All the data (past and coming) are available publicly in JSON:
 
 ### March
 
+* 4-5: [Gartner IAM Summit](https://www.gartner.com/) - London(UK)
```
Comment (scraly) [inline README.md:265]: Maybe change the URL to this one: https://www.gartner.com/en/conferences
?

File: README.md [hunk 2]
```diff
@@ -343,6 +360,14 @@ All the data (past and coming) are available publicly in JSON:
 
 ### October
 
+* 2: [IDM Europe](https://whitehallmedia.co.uk/future-events/) - Utrecht (Netherlands)
 * 7-11: [Devoxx Belgium](https://devoxx.com/#/) - Antwerp (Belgium)
 * 10-11: [Volcamp](https://www.volcamp.io) - Clermont-Ferrand (France)
+* 15: [CYBERGOV](https://whitehallmedia.co.uk/future-events/) - London (UK)
 * 27-29: [All Things Open](https://2023.allthingsopen.org/ato-2024-save-the-date/) - Raleigh (USA)
+
+### Novembre
```
Comment (scraly) [inline README.md:369]:
```suggestion
### November
```

**Review by scraly — CHANGES_REQUESTED** [review]
thanks
here some suggestions :-)

**Comment by mikamakusa** [comment]
Updates done, especially for the gartner conference for which i've forgotten to add the biggest part of the URL ;)

File: README.md [hunk 3]
```diff
@@ -256,11 +262,13 @@ All the data (past and coming) are available publicly in JSON:
 
 ### March
 
+* 4-5: [Gartner IAM Summit](https://www.gartner.com/en/conferences/emea/identity-access-management-uk?utm_campaign=Bimonthly%20newsletter&utm_medium=email&_hsmi=278991150&_hsenc=p2ANqtz--7CE7iGl0OVevnIDv0Ie5zXladlTsg4atF3lWIdIwIwdxNq5i2Jp9WdZe9FQI--OyDcyN6VV9_BtP0wuxha1Cct9V2tg&utm_content=278991150&utm_source=hs_email) - London(UK)
```
Comment (scraly) [inline README.md:265]:
```suggestion
* 4-5: [Gartner IAM Summit](https://www.gartner.com/en/conferences/emea/identity-access-management-uk) - London(UK)
```

**Review by scraly — CHANGES_REQUESTED** [review]
a final request change :)

File: README.md [hunk 4]
```diff
@@ -343,6 +360,15 @@ All the data (past and coming) are available publicly in JSON:
 
 ### October
 
+
```
Comment (scraly) [inline README.md:363]:
```suggestion
```

**Review by scraly — CHANGES_REQUESTED** [review]

**Comment by scraly** [comment]
merged :)

**Pull request merged**